packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -37,7 +37,7 @@ const app = createApp({
       viewTechDoc: techdocsPlugin.routes.docRoot,
     });
     bind(apiDocsPlugin.externalRoutes, {
-      createComponent: scaffolderPlugin.routes.root,
+      registerComponent: catalogImportPlugin.routes.importPage,
```

Comment:
Think we should create another `changeset` here for the changes in `create-app` and have a nice diff of what you have to do to your own Backstage instance here.

This change is a `patch` change, but you can take a look at some examples here for some nice changelog entries on `create-app` https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md

Reply:
Gave it a shot in `.changeset/friendly-weeks-cry.md`. Let me know if that seems right.